dags/pytorch_xla/vllm-nightly.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2023 Google LLC
```

Comment:
Are all these 4 tests doing the same thing? What is the difference between 2, 3, 4?

Reply:
thanks, this draft file has been deleted, same for the 2, 3, 4 too